Makefile

```diff
@@ -139,6 +140,18 @@ generate-clientset: .remove-clientset .remove-deepcopy .remove-crds
 .remove-clientset:
 	rm -rf $(CURDIR)/$(PKG_BASE)/clientset
 
+build-image: $(DOCKERFILE_DEVEL)
+	$(DOCKER) build \
+		--progress=plain \
+		--build-arg GOLANG_VERSION="$(GOLANG_VERSION)" \
+		--build-arg CLIENT_GEN_VERSION="$(CLIENT_GEN_VERSION)" \
+		--build-arg CONTROLLER_GEN_VERSION="$(CONTROLLER_GEN_VERSION)" \
+		--build-arg GOLANGCI_LINT_VERSION="$(GOLANGCI_LINT_VERSION)" \
+		--build-arg MOQ_VERSION="$(MOQ_VERSION)" \
+		--tag $(BUILDIMAGE) \
+		-f $(DOCKERFILE_DEVEL) \
+		.
```

Comment:
Do we need the build context to be the whole repo? Does it make sense to just make it `deployments/container`?

Reply:
the Dockerfile has no `COPY` step so it doesn't matter

Reply:
I would say it does matter because of this:
```
# We need to set the /work directory as a safe directory.
# This allows git commands to run in the container.
RUN git config --file=/.gitconfig --add safe.directory /work
```